heroku/cli
# refactor: migrate members:remove to oclif/core

[Work item
](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001aw25KYAQ/view)

## Testing
- check out branch and run yarn and yarn build
- Pick a team to test on (I like to use hk-front-end-online-test-team and hk-front-end-legacy-ent-team)
- Testing removal
    - Remove someone from the team using ./bin/run members:remove -t TEAM-NAME EMAIL
    - Make sure to add them back when done testing
- Testing revoking invite
    - hk-front-end-online-test-team is set up for this test
    - Invite someone to join the team
    - Revoke the invite using ./bin/run members:remove -t hk-front-end-online-test-team EMAIL
